src/amo/components/Addon/index.js

```diff
@@ -173,114 +185,174 @@ export class AddonBase extends React.Component {
+    }
 
-            <AddonMeta addon={addon} />
+    // eslint-disable react/no-danger
+    return (
+      <div className={classNames('Addon', `Addon-${addonType}`)}>
+        {errorBanner}
+        <Card className="" photonStyle>
+          <header className="Addon-header">
+            <section className="Addon-title-section">
+              <h1 className="Addon-title" {...titleProps}>
+                {titlePlaceholder}
+              </h1>
+              <p className="Addon-summary" {...summaryProps}>
+                {summaryPlaceholder}
+              </p>
+              {addon ?
+                <InstallButton
+                  {...this.props}
+                  className="Button--action Button--small"
+                  disabled={!isCompatible}
+                  ref={(ref) => { this.installButton = ref; }}
+                  status={installStatus}
+                /> : null
+              }
+            </section>
+
+            <section className="Addon-metadata">
+              {this.headerImage({ compatible: isCompatible })}
+
+              <h2 className="visually-hidden">
+                {i18n.gettext('Extension Metadata')}
+              </h2>
+              {addon ? <AddonMeta addon={addon} /> : <LoadingText />}
+            </section>
           </header>
 
-          {!compatible ? (
-            <AddonCompatibilityError maxVersion={maxVersion}
-              minVersion={minVersion} reason={reason} />
+          {compatibility && !isCompatible ? (
+            <AddonCompatibilityError
+              maxVersion={compatibility.maxVersion}
+              minVersion={compatibility.minVersion}
+              reason={compatibility.reason}
+            />
           ) : null}
         </Card>
 
         <div className="Addon-details">
-          {addon.previews.length > 0 ? (
+          {addonPreviews.length > 0 ? (
             <Card
               className="Addon-screenshots"
               header={i18n.gettext('Screenshots')}
             >
-              <ScreenShots previews={addon.previews} />
+              <ScreenShots previews={addonPreviews} />
             </Card>
           ) : null}
 
-          {description && description.length ? (
-            <ShowMoreCard
-              header={i18n.sprintf(
-                i18n.gettext('About this %(addonType)s'),
-                { addonType: addon.type }
-              )}
-              className="AddonDescription"
-            >
-              <div
-                className="AddonDescription-contents"
-                ref={(ref) => { this.addonDescription = ref; }}
-                dangerouslySetInnerHTML={
-                  sanitizeHTML(nl2br(description), allowedDescriptionTags)
-                }
-              />
-            </ShowMoreCard>
-          ) : null}
-
+          {this.renderShowMoreCard()}
           {this.renderRatingsCard()}
 
-          <AddonMoreInfo addon={addon} />
+          {addon ? <AddonMoreInfo addon={addon} /> : null}
```

Comment:
`AddonMoreInfo` was pretty tough to convert. I don't think it's worth the effort right now.